internals/scripts/validate.sh

```diff
@@ -0,0 +1,3 @@
+npm install --global @exodus/schemasafe
```

Comment:
I would move this step to the Dockerfile, so the Docker container already contains the dependency.